### ISSUE TITLE
Accessibility wrt color

### ISSUE THREAD

**Issue by melodykramer** [18F/guides-template#29]
From an email I received about the content guide:

I like the 18F content style guide very much.  I just wish it was easier to read.

I know that the Federal government is not required to follow WCAG 2.0 accessibility standards yet, but designers could still have some compassion for those of us with aging eyes.

The WCAG 2.0 guideline for contrast is a ratio of 4.5:1.  The contrast in the left navigation bar is 1.9:1.  The text is better but still difficult to read without effort.

This fad of minimalist sites with grey on white can't end soon enough for me.

**Comment by mbland** [comment]
cc: @nickbristow

**Comment by nickbristow** [comment]
@melodykramer which text is 1.9:1?

**Comment by melodykramer** [comment]
I think she's referring to the left navigation bar.

**Comment by nickbristow** [comment]
@melodykramer we should definitely consider making it darker if someone is having issues with it, I think the text is a little thin, but i'm not sure how they are getting 1.9:1. 

The text is `#74767B` which is `4.55:1` on white.

**Comment by jenniferthibault** [comment]
@maya also just pointed out in #27  that the hover color for links is not meeting the correct ratio.

The hover states are #7EB8DD over white backgrounds #FFFFFF, with a contrast ratio of 2.1:1
http://snook.ca/technical/colour_contrast/colour.html#fg=7EB8DD,bg=FFFFFF

**Comment by maya** [comment]
@jenniferthibault here is a darker shade of our `#18f` brand color which meets WCAG AA requirements: `#006fdd`. It's 10% darker than our current shade of blue.

<img width="454" alt="screen shot 2015-08-10 at 9 18 39 pm" src="https://cloud.githubusercontent.com/assets/5249443/9189997/17e7a76a-3fa5-11e5-9b99-312163f8bfe6.png">

<img width="894" alt="screen shot 2015-08-10 at 9 16 28 pm" src="https://cloud.githubusercontent.com/assets/5249443/9189998/1bdf0728-3fa5-11e5-8ff2-f62fc2cb90f9.png">

**Comment by maya** [comment]
I agree that our body font is hard read in general. I've always found it very thin and strains my eyes. 

Here are the guides set in Source Sans Pro as an example of an alternate font that's less wiry. I think it's important that the reading experience on the guides/site/etc is as readable and pleasant as possible. We don't want people bailing from our site bc they're having trouble or straining to read it. We should be designing for people with "aging eyes" first.

Source Sans Pro (new):
<img width="1046" alt="screen shot 2015-08-10 at 10 54 30 pm" src="https://cloud.githubusercontent.com/assets/5249443/9191208/357e3f24-3fb3-11e5-9b4c-958aa191696d.png">

Open Sans (current):
<img width="1049" alt="screen shot 2015-08-10 at 11 00 52 pm" src="https://cloud.githubusercontent.com/assets/5249443/9191220/5f134dfc-3fb3-11e5-8e10-ac30be2f719b.png">

**Comment by maya** [comment]
After a bit of digging, I think I found the culprit! We are using Open Sans 300, the light weight option of Open Sans. I switched to 400, the normal weight, and this is what it looks like:

<img width="1140" alt="screen shot 2015-08-11 at 12 11 23 am" src="https://cloud.githubusercontent.com/assets/5249443/9192174/62eb01ae-3fbd-11e5-83f5-a5d40957e1fc.png">

<img width="692" alt="screen shot 2015-08-11 at 12 10 12 am" src="https://cloud.githubusercontent.com/assets/5249443/9192188/8efb7b70-3fbd-11e5-8f88-1d20ee176194.png">

**Comment by mbland** [comment]
As someone whose eyes are aging (very slowly, but still), I endorse this change. :-) Care to send a PR?

**Comment by jenniferthibault** [comment]
Quick Q though @maya : why would you switch it from Open Sans to Source Sans? Seems like changing the weight would address the issue without changing the whole font family, which is used on the 18F site and other places as well (now print extensions into the Method Cards). That would be my vote, but maybe we should move this discussion to a new issue on typography, not color?

**Comment by maya** [comment]
@jenniferthibault that was just an example before I realized in https://github.com/18F/guides-template/issues/29#issuecomment-129736082 that it was actually a font weight problem, not font family.

**Comment by maya** [comment]
@jenniferthibault @mbland @melodykramer PR #32 sent!

**Comment by andrewmaier** [comment]
<3 @maya @jenniferthibault :)

**Comment by fureigh** [comment]
Was this fixed in #32 or is the contrast still an issue?

**Comment by maya** [comment]
@fureigh yes, it was fixed.

**Comment by fureigh** [comment]
@maya :+1:
